js/app/data-artists.js

```diff
@@ -24,9 +24,11 @@ export const blogDataArtist = dataArtist({
     author: ["elements", "author", "value", 0, findAuthorName],
     title: ["elements", "title", "value"],
     posted: ["elements", "post_date", "value", formatDate],
+    "posted-datetime": ["elements", "post_date", "value"],
```

Comment:
`posted` and `posted-datetime` are a little confusing to me here. Should these be swapped around?

Also, `posted_datetime` would be more consistent with other naming used.

Reply:
"posted" is the human-readable formatted date for display
"posted-datetime" is the ISO8601 date _and time_ (aka timestamp) for inclusion in the time element (as discussed in the blog: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-time-element:valid-global-date-and-time-string-2)

The underscore syntax of "post_date" is something which is part of the data structure controlled by Kentico and doesn't conform to de facto industry standards such as https://github.com/airbnb/javascript#naming-conventions.

Reply:
perhaps refactoring this structure so that it returned an object with both the human readable and timestamp versions agains `posted` would be more readable.

Reply:
> perhaps refactoring this algorithm so that it returned an object with both the human readable and timestamp versions agains `posted` would be more readable?

I like this idea

Reply:
> The underscore syntax of "post_date" is something which is part of the data structure controlled by Kentico and doesn't conform to de facto industry standards such as https://github.com/airbnb/javascript#naming-conventions.

With this I was referring to `posted-datetime` as you have used underscore elsewhere:

```
export const pagingArtist = dataArtist({
    next: ["pagination", "next_page"],
    count: ["pagination", "count"],
    total_count: ["pagination", "total_count"],
    page: ["pagination", calculateCurrentPage]
});
```
So as to keep consistent...

Reply:
aha - you are right. It appears I was torn between the API and javascript naming conventions. Good spot!

Reply:
> > perhaps refactoring this algorithm so that it returned an object with both the human readable and timestamp versions agains `posted` would be more readable?
> 
> I like this idea

I'm going to tackle this in a separate PR because this one already has 8 files in it...

Reply:
> aha - you are right. It appears I was torn between the API and javascript naming conventions. Good spot!

done